package.json

```diff
@@ -16,6 +16,7 @@
     "start": "node server.js",
     "clean": "rimraf dist",
     "build": "npm run clean && npm run build:webpack",
+    "build:dev": "webpack --config webpack.config.js",
```

Comment:
There's still quite a few ways to improve build time, but I have yet tried, something related to DLL